SmartDeviceLinkTests/DevAPISpecs/SDLLifecycleManagerSpec.m

```diff
@@ -430,21 +429,21 @@ @interface SDLLifecycleManager ()
                     registerAppInterfaceResponse.resultCode = SDLResultWrongLanguage;
                     registerAppInterfaceResponse.info = @"Language mismatch";
                     registerAppInterfaceResponse.language = SDLLanguageEnGb;
+                    registerAppInterfaceResponse.hmiDisplayLanguage = SDLLanguageEnGb;
                     testManager.registerResponse = registerAppInterfaceResponse;
 
                     SDLLifecycleConfigurationUpdate *update = [[SDLLifecycleConfigurationUpdate alloc] initWithAppName:@"EnGb" shortAppName:@"E" ttsName:[SDLTTSChunk textChunksFromString:@"EnGb ttsName"] voiceRecognitionCommandNames:nil];
-                    OCMStub([testManager.delegate managerShouldUpdateLifecycleToLanguage:[OCMArg any]]).andReturn(update);
+                    OCMStub([testManager.delegate managerShouldUpdateLifecycleToLanguage:[OCMArg any] hmiLanguage:[OCMArg any]]).andReturn(update);
```

Comment:
The older delegate method also needs to continue to be tested. So a new test should be added but the old test needs to continue to exist.